server/CGameHandler.h

```diff
@@ -309,6 +310,7 @@ class CGameHandler : public IGameCallback, public CBattleInfoCallback, public En
 
 		const CGHeroInstance *winnerHero, *loserHero;
 		PlayerColor victor, loser;
+		ui8 winnerSide;

```

Comment:
Just in case sometimes it is possible to have draw when both sides loose

Reply:
Action item was added